benchmarks/benchmarks/fft_basic.py

```diff
@@ -103,6 +103,25 @@ def time_ifft(self, size, cmplx, module):
         self.ifft(self.x)
 
 
+class NextFastLen(Benchmark):
+    params = [
+        [12, 13,  # small ones
+         1021, 1024,  # 2 ** 10 and a prime
+         16381, 16384,  # 2 ** 14 and a prime
+         262139, 262144,  # 2 ** 17 and a prime
+         999983, 1048576,  # 2 ** 20 and a prime
+         ],
+    ]
+    param_names = ['size']
+
+    def setup(self, size):
+        if not has_scipy_fft:
+            raise NotImplementedError
+
+    def time_next_fast_len(self, size):
+        scipy_fft.next_fast_len(size)
```

Comment:
Isn't this benchmark essentially pointless now? Perhaps it would be useful to add cases for `next_fast_len.__wrapped__` and for calling `next_fast_len.cache_clear()` between calls (If that can be done without timing the clear operation itself).